src/components/VotingOptionTag/VotingOptionTag.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+	import { DefaultVoteOption, type CustomVoteOption } from '$models/voting';
+	import { Tag } from 'carbon-components-svelte';
+
+	export let voteOption: DefaultVoteOption | CustomVoteOption;
+
+	let className = '';
+	export { className as class };
+
+	let tagColorClass = 'bg-purple-70';
+	let tagBgOpacity = 1;
+	let label = '';
+
+	if (voteOption === DefaultVoteOption.Agreed) {
```

Comment:
When we compare several cases on the same variable, `switch` is easier to read, write, and maintain, because we don't need to repeat `voteOption ===` in every condition (that's why it used to be that way, sorry that I'm not clear but my point is we don't need "both")

I suggest you use the switch on `voteOption`, and put everything in the `else` inside `default:` which will give the same result.

```ts
switch(voteOption):
  case DefaultVoteOption.Agreed:
  // ...
  default:
  // code in else
```

Reply:
Oh, I see. I didn't think about that. Going to revert and fix it.